src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,63 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manSer =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService drSer =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carSer =
+            (CarService) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+
+        Driver bob = drSer.get(5L);
+        Driver john = drSer.get(6L);
+        Driver alice = drSer.get(7L);
+
+        List<Driver> drivers = new ArrayList<>();
+        drivers.add(bob);
+        drivers.add(john);
+        drivers.add(alice);
+
+        Driver ivan = new Driver();
+        ivan.setName("Ivan");
```

Comment:
You are directly setting values to the 'Driver' object. It's better to use a constructor or a builder pattern to initialize an object.